apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/GetSchemaTableTests.cs

```diff
@@ -32,42 +33,79 @@ public async Task GetSchemaTable_Default_ReturnsNull()
         {
             using (var connection = _fixture.GetConnection())
             {
-                var command = connection.CreateSelectCommand($"SELECT Int64Value, BytesArrayValue FROM {_fixture.TableName}");
+                var command = connection.CreateSelectCommand($"SELECT * FROM {_fixture.TableName}");
                 using (var reader = await command.ExecuteReaderAsync())
                 {
                     Assert.Null(reader.GetSchemaTable());
                 }
             }
         }
 
-        [Fact]
-        public async Task GetSchemaTable_WithFlagEnabled_ReturnsNull()
+        public static List<object[]> SchemaTestData =>
+            new List<object[]>
+            {
+                // Base types.
+                new object[] { "BoolValue", typeof(bool), SpannerDbType.Bool },
+                new object[] { "Int64Value", typeof(long), SpannerDbType.Int64 },
+                new object[] { "Float64Value", typeof(double), SpannerDbType.Float64 },
+                new object[] { "NumericValue", typeof(SpannerNumeric), SpannerDbType.Numeric },
+                new object[] { "StringValue", typeof(string), SpannerDbType.String },
+                new object[] { "BytesValue", typeof(byte[]), SpannerDbType.Bytes },
+                new object[] { "TimestampValue", typeof(DateTime), SpannerDbType.Timestamp },
+                new object[] { "DateValue", typeof(DateTime), SpannerDbType.Date },
+                // Array types.
+                new object[] { "BoolArrayValue", typeof(List<bool>), SpannerDbType.ArrayOf(SpannerDbType.Bool) },
+                new object[] { "Int64ArrayValue", typeof(List<long>), SpannerDbType.ArrayOf(SpannerDbType.Int64) },
+                new object[] { "Float64ArrayValue", typeof(List<double>), SpannerDbType.ArrayOf(SpannerDbType.Float64) },
+                new object[] { "NumericArrayValue", typeof(List<SpannerNumeric>), SpannerDbType.ArrayOf(SpannerDbType.Numeric) },
+                new object[] { "StringArrayValue", typeof(List<string>), SpannerDbType.ArrayOf(SpannerDbType.String) },
+                new object[] { "BytesArrayValue", typeof(List<byte[]>), SpannerDbType.ArrayOf(SpannerDbType.Bytes) },
+                new object[] { "TimestampArrayValue", typeof(List<DateTime>), SpannerDbType.ArrayOf(SpannerDbType.Timestamp) },
+                new object[] { "DateArrayValue", typeof(List<DateTime>), SpannerDbType.ArrayOf(SpannerDbType.Date) },
+            };
+
+        [MemberData(nameof(SchemaTestData))]
+        [Theory]
```

Comment:
Nit: please reverse these. We always have Theory/Fact at the top, followed by any member data.

Reply:
Done.